webpack/assets/javascripts/react_app/components/common/SkeletonLoader/DefaultLoaderEmptyState.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { translate as __ } from '../../../common/I18n';
+import './styles.scss';
+
+const DefaultLoaderEmptyState = () => (
+  <span className="disabled-text">{__('Not Available')}</span>
```

Comment:
Do we really need another file for a one-line component that is only used in one place?

Reply:
we use here a specific css file here, and define this span under `SkeleotonLoader` component will re-create it on every re-render.